### DISCUSSION TITLE
I accidently pressed a key and now this recipe is pinned and i cant seem to unpin it

### DISCUSSION THREAD

**Discussion by walksanatora** [shedaniel/RoughlyEnoughItems#1243] [Q & A] [answered]
I know it has to invomve the R key somehow, but idk which other buttons to press
![image](https://user-images.githubusercontent.com/47339836/204053026-bb8dc182-2653-4373-a8c9-d3a026fc40b8.png)

**Comment by rozbrajaczpoziomow** [comment, marked as answer]
A
